task.go

```diff
@@ -104,7 +106,7 @@ func RunTask(name string) error {
 	}
 
 	for i := range t.Cmds {
-		if err = t.runCommand(i); err != nil {
+		if err = t.runCommand(i, t.Env); err != nil {
```

Comment:
Don't need to pass `Env` as parameter. Task is available as a receiver of the func.